task-manager/src/views/LoginView.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <v-container
+        fluid
+        class="d-flex align-center justify-center login-container"
+    >
+        <v-card class="elevation-12 login-card">
+            <v-card-title class="headline">Login</v-card-title>
+            <v-card-text>
+                <v-text-field
+                    v-model="username"
```

Comment:
can we have a implementation that if the user adds his email address in capital then also it should work
also lets focus the mouse on the username input as soon as the page is loaded

Reply:
Okay, I will add this.